packages/aws-cdk/lib/init-templates/app/java/cdk.json

```diff
@@ -1,3 +1,3 @@
 {
-  "app": "mvn -e -q exec:java"
+  "app": "mvn compile -q && mvn -e -q exec:java"
```

Comment:
```suggestion
  "app": "mvn -e -q compile exec:java"
```

Reply:
Figured there'd be a way to simplify it!